miniwallet/deploy/004_deploy_mini1155.ts

```diff
@@ -0,0 +1,111 @@
+import config from '../config'
+import { HardhatRuntimeEnvironment } from 'hardhat/types'
+import { DeployFunction } from 'hardhat-deploy/types'
+import { ethers } from 'hardhat'
+
+const deployFunction: DeployFunction = async function (
+  hre: HardhatRuntimeEnvironment
+) {
+  const { deployments, getNamedAccounts, getChainId } = hre
+  const { deploy } = deployments
+  const { deployer, operatorA } = await getNamedAccounts()
+  const chainId = await getChainId()
+  const mini1155DeployArgs = {}
+  let saleIsActive
+  let metadataFrozen
+  let mintPrice
+  let maxPerMint
+  let standardTokenId
+  let rareTokenId
+  let exchangeRatio
+  let rareProbabilityPercentage
+  let salt
+  let baseUri
+  let contractUri
+
+  console.log(`chainId: ${chainId}`)
+  if (chainId === '1666600000,') {
```

Comment:
same issue

Reply:
Refactored approach and commit are [here](https://github.com/polymorpher/sms-wallet/issues/13#issuecomment-1246191757)